packages/UI/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  ChevronsLeft,
+  ChevronsRight,
+  ChevronRight,
+  ChevronLeft,
+} from 'lucide-react';
+import Link from 'next/link';
+import * as React from 'react';
+
+import type { VariantProps } from 'tailwind-variants';
+import { tv } from 'tailwind-variants';
+import { cn } from '../../utils/classesMerger';
+
+const Pagination = ({ className, ...props }: React.ComponentProps<'nav'>) => (
```

Comment:
Is there any standard on using function over using a const variable for a component?  I see forwardRef needs to be this pattern

Reply:
yeah I prefer using const personally but we don't restrict devs on using it, maybe I should make eslint warn users for not using `const` way